package.json

```diff
@@ -53,7 +53,7 @@
     "tsyringe": "~4.6.0",
     "win-ca": "^3.5.0",
     "ws": "~8.1.0",
-    "yargs": "~16.2.0"
+    "yargs": "^17.7.2"
```

Comment:
The bump of any dependencies is not the aim and purpose of your task; revert redundant changes if you do not have any good reason to keep it.

Reply:
Hi @derevnjuk, I've upgraded to 17 version to have of async command builder, as stated in the [issue](https://github.com/NeuraLegion/bright-cli/issues/493) description:
>In order to keep all the behavior async command builder has to be implemented. It requires to bump yargs to v17.

Reply:
I see. In order to keep this PR manageable and provide an easier way to revert changes if needed, let's open a separate PR to handle yargs bump only